### PR TITLE
EOS-26026: Deploy Utils REST Server in K8s env

### PR DESCRIPTION
**Pull request by suryakumar1024**
# Problem Statement
- Deploy Utils REST Server in K8s environment and expose the container service in 28300 port

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for the design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered experiments/logging/glusterfs/README.md](https://github.com/suryakumar1024/cortx-prvsnr/blob/26026_utils_server/experiments/logging/glusterfs/README.md)
[View rendered experiments/logging/local/README.md](https://github.com/suryakumar1024/cortx-prvsnr/blob/26026_utils_server/experiments/logging/local/README.md)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: test/deploy/kubernetes/service.sh [hunk 1]
```diff
@@ -87,6 +87,8 @@ fi
 # Create Control RunTime Node (POD)
```
Comment (nitin-seagate) [inline test/deploy/kubernetes/service.sh:87]: Apply message service before creating control pod... Infact even before creating HA node...
Also please handle deletion case for message service

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: test/deploy/kubernetes/service.sh (hunk 1 above)
Comment (nitin-seagate) [reply test/deploy/kubernetes/service.sh:87]: Line No. 114 change message as:
`print_header "Creating Storage RunTime Node - $NODE_NAME";`

Comment (nitin-seagate) [reply test/deploy/kubernetes/service.sh:87]: Line No. 88 change message as:
`print_header "Creating Control RunTime Node - Cluster";`

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by tanujashinde0405 — APPROVED** [review]

File: test/deploy/kubernetes/service.sh (hunk 1 above)
Comment (sachinpunadikar) [reply test/deploy/kubernetes/service.sh]: Removed the unwanted code

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**